src/Classes/CommentDBConnector/CommentSQLDBConnector/index.ts

```diff
@@ -0,0 +1,169 @@
+import { PoolClient } from 'pg';
+import { getConnector } from '../../../Connect/sqlDBConnector';
+import CommentData from '../../../Interfaces/Data/Comment';
+import CommentDBConnector from '../../../Interfaces/DBConnectors/CommentDBConnector';
+import SQLDBConnector from '../../DBConnector/SQLDBConnector';
+
+class CommentSQLDBConnector extends SQLDBConnector
+  implements CommentDBConnector {
+  connector: PoolClient;
+  constructor() {
+    super();
+    this.connector = getConnector();
+    this.createTables();
+  }
+
+  async createTables(): Promise<void> {
+    try {
+      await this.connector.query('BEGIN');
+      await this.connector.query(
+        `CREATE TABLE IF NOT EXISTS COMMENTS(
+            comment_id uuid PRIMARY KEY,
```

Comment:
please replace uuid with type SERIAL with autoincrementing. Note: replace it everywhere instead of uuid